watson/cli.py

```diff
@@ -1269,6 +1269,8 @@ def edit(watson, confirm_new_project, confirm_new_tag, id):
             if not watson.is_started and start > stop:
                 raise ValueError(
                     "Task cannot end before it starts.")
+            if start > arrow.utcnow():
+                raise ValueError("Start time cannot be in the future")
```

Comment:
Maybe we should also check that the stop time is not in the future?

Reply:
Right. I will add that and update the changelog, forgot about that.